### PR TITLE
Add array and cli/py tests for querying records with null attributes

### PR DESCRIPTION
**Pull request by aaronwolen**
Adds cli/py tests for testing against bug fixed in #142

### CODE REVIEW

File: apis/python/tests/test_tiledbvcf.py [hunk 1]
```diff
@@ -372,7 +372,9 @@ def test_read_null_attrs(tmp_path):
                                   np.array([89], dtype=np.int32), None,
                                   np.array([24], dtype=np.int32), None, None]),
             'fmt_DP': pd.Series([0, 0, 15, 64, 6, 2, 180,
-                                     88, 97, 24, 23, 21], dtype=np.int32)})
+                                     88, 97, 24, 23, 21], dtype=np.int32),
+            'fmt_MIN_DP': pd.Series([0, 0, 14, 30, 3, 1, 20,
+                                     None, 24, None, 23, 19], dtype=np.int32)})
```
Comment (Shelnutt2) [inline apis/python/tests/test_tiledbvcf.py:377]: I think you need to change the dtype to `"Int32"` or `pd.Int32Dtype()`, per https://pandas.pydata.org/pandas-docs/stable/user_guide/integer_na.html

Comment (aaronwolen) [reply apis/python/tests/test_tiledbvcf.py]: Updated the `dtype` as discussed and noted issue in ch2928.
